airflow/jobs/scheduler_job.py

```diff
@@ -785,8 +785,7 @@ def process_file(self, file_path, zombies, pickle_dags=False, session=None):
             return [], len(dagbag.import_errors)
 
         # Save individual DAGs in the ORM and update DagModel.last_scheduled_time
-        for dag in dagbag.dags.values():
-            dag.sync_to_db()
+        DAG.bulk_sync_to_db(dagbag.dags.values())
```

Comment:
I think this would should live on DagBag:

```suggestion
        dagbag.bulk_sync_to_db()
```

Reply:
I created a helper method in DagBag.